app/models/__init__.py

```
@@ -21,8 +21,10 @@
 # from .products import Product, ProductCategory
 # from .orders import Order, OrderItem
 # from .shared import Location
+from .farmers import Farmer, FarmerVerification
```

Flake8 (7.2.0)

[error] '.farmers.FarmerVerification' imported but unused

(F401)